.github/workflows/jira.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         else
           echo "Actor ${{ github.actor }} is not a ${TEAM} team member"
         fi
-        echo "::set-output name=role::${ROLE}"
+        echo "role=${ROLE}" >> $GITHUB_OUTPUT
       env:
         GITHUB_TOKEN: ${{ secrets.JIRA_SYNC_GITHUB_TOKEN }}
```

Comment:
Unrelated but I assume it's broken bc the token is incorrect. I wonder if we can just use the `${{ secrets.GITHUB_TOKEN }}` or if it actually needs extra permissions to get the team membership

Reply:
I think you are correct about their being an issue with the token! According to [these docs](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api?apiVersion=2022-11-28#:~:text=Instead%20of%20using%20the%20gh%20auth%20login%20command%2C%20pass%20your%20token%20as%20an%20environment%20variable%20called%20GH_TOKEN) github does recommend on using `GITHUB_TOKEN` in order to authenticate to use the `hub api`. I also think the main issue could be with the env variable name because in the docs the name is `GH_TOKEN` not `GITHUB_TOKEN`.